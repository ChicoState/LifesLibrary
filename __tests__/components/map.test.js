import React from 'react';
import renderer from 'react-test-renderer';

import Map from '../../components/map.js';

it('render component scanner', () => {
  const TextInputComponent = renderer.create(<Map />).toJSON();
  expect(TextInputComponent).toMatchSnapshot();
});
