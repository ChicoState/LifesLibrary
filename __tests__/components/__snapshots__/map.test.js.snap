// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render component scanner 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <AIRMap
    handlePanDrag={false}
    initialRegion={
      Object {
        "latitude": 39.7285,
        "latitudeDelta": 0.0922,
        "longitude": -121.8375,
        "longitudeDelta": 0.0421,
      }
    }
    onChange={[Function]}
    onLayout={[Function]}
    onMapReady={[Function]}
    onMarkerPress={[Function]}
    showsMyLocationButton={true}
    showsUserLocation={true}
    style={
      Object {
        "height": 1334,
        "width": 750,
      }
    }
  />
</View>
`;
